File: src/com/krishna/miniproject/StudentUser.java
package com.krishna.miniproject;

class StudentUser extends User{
    private String course;

    public StudentUser(String name, String userId, String course) {
        super(name, userId);
        this.course = course;
    }
    @Override
    public void showDetails(){
        System.out.println("Name: "+ name + "\nUser Id: "+ userId + "\nCourse: "+course);
    }
}

File: Dart/src/krishna/miniproject/StudentUser.dart
import 'User.dart';

class StudentUser extends User{

  String _course;

  StudentUser(String name, String userId,String course) :
        _course=course,
        super(name,userId);

  @override
  void showDetails(){
    print("Name: $name");
    print("User Id: $userId");
    print("Course: $_course");
  }


}